app/models/planet.py

```diff
@@ -0,0 +1,17 @@
+class Planet:
+    def __init__(self, id, name, description):
+        self.id = id
+        self.name = name
+        self.description = description
+
```

Comment:
How would you write the `to_dict` instance method so you can take an instance of Planet and turn it into a dictionary that will be returned as a response to a client's request?